site/content/docs/0.0/utilities/shadows.md

```diff
@@ -8,4 +8,71 @@ aliases:
 toc: true
 ---
 
-{{< callout-soon "page" >}}
+{{< callout info >}}
+Box-shadow styles are referred to as elevation in the design system.
+{{< /callout >}}
+
+## Examples
+
+While shadows on components are disabled by default in OUDS Web and can be enabled via `$enable-shadows`, you can also quickly add or remove a shadow with our `box-shadow` utility classes. Includes support for `.shadow-none` and new semantic helpers.
```

Comment:
> While shadows on components are disabled by default in OUDS Web and can be enabled via `$enable-shadows`,

We don't have components yet. So either we remove this sentence, or either we mention `$enable-shadows` in the documentation (other parts of the docs should be uncommented)